.travis.yml

```diff
@@ -4,3 +4,9 @@ node_js:
   - '10'
   - '12'
 script: ./script/travis_run_js_tests.sh
```

Comment:
We should make sure we delete the credentials before running any tests here.
There are known vulnerabilities exploiting credentials that are (unnecessarily) available in ci environments.